lib/datadog/appsec/remote.rb

```diff
@@ -63,18 +65,20 @@ def receivers
             overrides = []
             exclusions = []
 
-            asm_data_config_types = ['blocked_ips', 'blocked_users']
-            asm_overrides_config_types = ['blocking', 'disabled_rules']
-
             repository.contents.each do |content|
               case content.path.product
               when 'ASM_DD'
                 rules << parse_content(content)
               when 'ASM_DATA'
-                data << parse_content(content) if asm_data_config_types.include?(content.path.config_id)
+                data << parse_content(content) if ASM_DATA_CONFIG_TYPES.include?(content.path.config_id)
```

Comment:
ASM_DATA files need further merging, as explained [here](https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2846623263/ASM+DATA+-+Blocked+IP+and+User+Data). Is it done somewhere else?

Reply:
seems like we miss at least `custom_rules` on the ASM schema as well. The hotfix won't cut it: we need to audit this deeper.